wrappers/kn/README.md

```diff
@@ -40,6 +42,48 @@ barcodeReader.read(image).joinToString("\n") { barcode: Barcode ->
 
 Here you have to load your image into memory by yourself and pass the decoded data to the constructor of `ImageView`.
 
+### Writing
+
+A trivial use case looks like this:
+
+```kotlin
+import zxingcpp.BarcodeFormat
+import zxingcpp.BarcodeWriter
+import zxingcpp.CreatorOptions
+import zxingcpp.ImageFormat
+import zxingcpp.ImageView
+import zxingcpp.toSVG
+import zxingcpp.toImage
```

Comment:
Those symbols don't exist, do they? I believe a `import zxingcpp.Barcode` is missing, though. And that would also be missing in the reading use case above?

Question: did you check that this code compiles? (I did not).

Reply:
Rewritten example code and compiled: https://github.com/zxing-cpp/zxing-cpp/pull/764/commits/a2cea2de74e1e8dd6b9d29812b3f7851e393f710